### PR TITLE
Perform operations on UTC time instead of using ActiveSupport::TimeZone

### PR DESCRIPTION
**Pull request by jcampos8782**

### CODE REVIEW

File: spec/ozone_spec.rb [hunk 1]
```diff
@@ -192,14 +217,14 @@ module Ozone
 
         fit 'converts a utc time to time with time zone and moves clocks back an hour' do
```
Comment (l8nite) [inline spec/ozone_spec.rb]: fit

File: lib/ozone.rb [hunk 2]
```diff
@@ -7,55 +7,31 @@ module Formatter
...
     end
 
     def before?(time_with_zone)
-      adjusted_time < time_with_zone
+      adjusted_time < to_ozone_time(time_with_zone).send(:adjusted_time)
     end
 
     def after?(time_with_zone)
-      adjusted_time > time_with_zone
+      adjusted_time > to_ozone_time(time_with_zone).send(:adjusted_time)
```
Comment (hubert) [inline lib/ozone.rb]: what if you make adjusted_time protected?

File: lib/ozone.rb [hunk 3]
```diff
@@ -64,11 +40,23 @@ def strftime(format: '%Y-%m-%d %H:%M')
 
     private
 
+    PST = ActiveSupport::TimeZone['Pacific Time (US & Canada)']
```
Comment (hubert) [inline lib/ozone.rb]: are you looking for any time zone that observes daylight savings?

Comment (jcampos8782) [inline lib/ozone.rb]: Exactly. I added a comment in the updated diff/PR

File: lib/ozone.rb (hunk 2 above)
Comment (jcampos8782) [inline lib/ozone.rb]: Done

File: spec/ozone_spec.rb (hunk 1 above)
Comment (jcampos8782) [inline spec/ozone_spec.rb]: Fixed

**Pull request merged**